cms/utils/conf.py

```diff
@@ -82,6 +82,7 @@ def wrapper():
     'PAGE_WIZARD_CONTENT_PLUGIN': 'TextPlugin',
     'PAGE_WIZARD_CONTENT_PLUGIN_BODY': 'body',
     'PAGE_WIZARD_CONTENT_PLACEHOLDER': None,  # Use first placeholder it finds.
+    'PAGETREE_DESCENDANTS_LIMIT': float('inf'),
```

Comment:
What is the reason for a float vs an integer?

Reply:
It is just a way to set this limit to `Infinity` by default. I cannot use `int('int')` to achieve that.

Reply:
Did you prefer to set this settings to `None` by default ?